app.js

```diff
@@ -10,14 +10,22 @@ if (argv._[0] === 'reset') {
   linter({ argv });
 } else if (argv._[0] === 'upgrade') {
   upgrader({ argv });
-} else {
+} else if (argv._[0] === 'help' || argv.help) {
   console.log(stripIndent`
-    Usage:
-
-    # List changes you need to make (always recommended)
-    apos-code-upgrader lint
+    Commands:
 
-    # Make some changes automatically (does not do everything)
-    apos-code-upgrader upgrade
+      lint                  List changes you need to make (always recommended)
+        Example: apos-code-upgrader lint
+      upgrade [options]     Make some changes automatically
+        Example: apos-code-upgrader upgrade
+        Options:
+        - --upgrade-required-files: Experimental support for inlining module
+          code included with \`require\` statements. Most successful when the
```

Comment:
Are we feeling these backticks are helpful because the developer's eye knows markdown, or would it make more sense to use single or double quotes for familiarity?

Reply:
That was my thinking. I'm open to either.

Reply:
A+